### PR TITLE
JWPLAYER-8: No space between 2 JW players

### PR DESCRIPTION
**Pull request by msmeria**

### CODE REVIEW

File: src/main/resources/Macros/JWPlayer.xml [hunk 1]
```diff
@@ -217,6 +218,120 @@ require(['jQueryNoConflict', 'jwplayer'], function($, jwplayer) {
...
+      <contentType>CSS</contentType>
+    </property>
+    <property>
+      <name>Style</name>
+    </property>
+    <property>
+      <parse/>
+    </property>
+    <property>
+      <use>always</use>
```
Comment (mflorea) [inline src/main/resources/Macros/JWPlayer.xml]: Loading this CSS on every page of the wiki is not nice.

File: src/main/resources/Macros/JWPlayer.xml [hunk 2]
```diff
@@ -217,6 +218,120 @@ require(['jQueryNoConflict', 'jwplayer'], function($, jwplayer) {
...
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <property>
+      <cache>long</cache>
+    </property>
+    <property>
+      <code>.jwplayer.playlist-none {
```
Comment (mflorea) [inline src/main/resources/Macros/JWPlayer.xml]: What if the player has a playlist?

**Comment by mflorea** [comment]
Some of the changes are not relevant for the issue you're trying to fix. Would be nice to filter them out.

File: src/main/resources/Macros/JWPlayer.xml (hunk 1 above)
Comment (msmeria) [inline src/main/resources/Macros/JWPlayer.xml]: This is the only setting that applied the CSS rule for a test page. The "On demand only" & "On this page or on demand" options didn't have any effect. Maybe this is a platform bug?

**Comment by msmeria** [comment]
@mflorea If you are talking about the `<editor>PureText</editor>` and `<editor/>` pieces of text, I think these were automatically added when I did a `mvn xar:format`. Should I remove them?

File: src/main/resources/Macros/JWPlayer.xml (hunk 1 above)
Comment (mflorea) [inline src/main/resources/Macros/JWPlayer.xml]: There's no bug. "On demand" means you need to include the CSS explicitly where you know it's needed. In your case this means "when the JWPlayer macro is used". You may want to read the [Skin Extension Tutorial](http://platform.xwiki.org/xwiki/bin/view/DevGuide/SkinExtensionsTutorial).

**Comment by mflorea** [comment]
``<editor>PureText</editor>`` comes from the XAR export not from the ``mvn xar:format``, because you exported from a recent version of XWiki where the JSX class has suffered some small changes. But these changes are not relevant for the issue you are fixing. You can add only the relevant changes using ``git add -i`` (interactive add). At most, you can do two commits, one that add irrelevant / formatting changes and then a second commit with the relevant changes.

**Comment by mflorea** [comment]
But this is minor (don't block the PR). You should focus on the in-line comments I made.

**Comment by msmeria** [comment]
@acotiuga I've fixed this in https://github.com/xwiki-contrib/macro-jwplayer/pull/3/commits/3da3eb3587b5f4a181aa6f70fd2b8f244aa61a5d#diff-d085921965eec6a2c530190cd48e4ef8R447

**Pull request merged**